torch_xla/csrc/ops/randperm_out.cpp

```diff
@@ -0,0 +1,38 @@
+#include "torch_xla/csrc/ops/randperm_out.h"
+
+#include "torch_xla/csrc/lowering_context.h"
+#include "tensorflow/compiler/xla/xla_client/xla_util.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/random.h"
+#include "torch_xla/csrc/torch_util.h"
+
+namespace torch_xla {
+
+// xw32: should I include a xla::Shape param in the constructor?
+// xw32: what opkind should I use? randperm or randperm_out? randperm_out doesn't seem to exist.
+// xw32: what does the shape param in the constructor mean? The output shape?
+// xw32: what does the hash in the constructor mean?
+RandpermOut::RandpermOut(int64_t n)
+    : XlaNode(torch::lazy::OpKind(at::aten::randperm), torch::lazy::OpList(),
+              xla::ShapeUtil::MakeShape(xla::U64, {n}),
```

Comment:
Why did you choose `xla::U64`?

Reply:
What's the guidelines to choose the element type, considering it's the element type of a shape?